Gopkg.toml

```diff
@@ -8,23 +8,23 @@
 
 [[constraint]]
   name = "github.com/grpc-ecosystem/go-grpc-middleware"
-  version = "1.0.0"
+  version = ">=1.0.0"
 
 [[constraint]]
   branch = "master"
   name = "github.com/lib/pq"
 
 [[constraint]]
   name = "github.com/grpc-ecosystem/grpc-gateway"
-  version = "1.4.1"
+  version = ">=1.13.0"
```

Comment:
can we have `>=1.4.1` here? having `>=1.13.0` may cause issues in some of existing apps which use lower version

Reply:
fixed